modules/installation-requirements-user-infra.adoc

```diff
@@ -173,7 +183,9 @@ On your z/VM instances, set up:
 [discrete]
 === Disk storage for the z/VM guest virtual machines
 
-* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. If available, use HyperPAV to ensure optimal performance.
+* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs.
+ifdef::ibm-z[To reach the minimum required DASD size for RHCOS installations, you need extended address volumes (EAV).]
```

Comment:
Same question as above on whether this ifdef is necessary.

Reply:
Same, removed.